### ISSUE TITLE
Can we expect Support for PBIP 

### ISSUE THREAD

**Issue by Rob-analyst-87** [pbi-tools/pbi-tools#303]
Hi @mthierba , i have been using PBI tools for my power BI projects. It helped me in many use cases, which is not possible otherwise. 
I want to see if PBI Tools support PBIP format.
I would like to use it to convert PBIX/PBIT to PBIP and other way.  Last, I heard you are working on it to support it in Reid Havens Video.  Just checking if you can share any timelines on the same.

Thanks for the great tool and your help to the PBI community is greatly appreciated.

**Comment by Rob-analyst-87** [comment]
@mthierba  any light on this ?

**Comment by mthierba** [comment]
I can't give an exact date at this point. I'm very busy with other projects (and pbi-tools doesn't pay my bills). I'm halfway through the necessary work, though.

**Comment by flanakin** [comment]
+1 eager to see this!

**Comment by jakeduddy** [comment]
Agreed. Would be nice to be able save a git on top of pbip to be in step with Microsoft and make new developer experience easier. But then have all the power and flexibility of pbitools for deployment. Rui has a Power Shell example which is very convoluted, and the git integration is a bit basic

**Comment by mloetscher** [comment]
+1 what's missing until this feature could be released?
